src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -334,14 +334,9 @@ namespace ExhaustAirSystemManager {
         }
     }
 
-    void InitExhaustAirSystem([[maybe_unused]] int &ExhaustAirSystemNum) // maybe unused
-    {
-    }
-
```

Comment:
[src/EnergyPlus/ExhaustAirSystemManager.cc:337]:(style),[unusedFunction],The function 'InitExhaustAirSystem' is never used.